stl/inc/xstring

```diff
@@ -1285,6 +1289,25 @@ public:
     constexpr basic_string_view(_It _First, _Se _Last) noexcept(noexcept(_Last - _First)) // strengthened
         : _Mydata(_STD to_address(_First)), _Mysize(static_cast<size_type>(_Last - _First)) {}
     // clang-format on
+
+#if _HAS_CXX23
+    // clang-format off
+    template <_RANGES contiguous_range _Range>
+        requires _RANGES sized_range<_Range>
+            && same_as<_RANGES range_value_t<_Range>, _Elem>
+            && (!is_convertible_v<_Range, const _Elem*>)
+            && (!requires(remove_reference_t<_Range>& _Rng) {
+                _Rng.operator basic_string_view<_Elem, _Traits>();

```

Comment:
```suggestion
            && (!requires(remove_cvref_t<_Range>& _Rng) {
                _Rng.operator _STD basic_string_view<_Elem, _Traits>();
```

This is one of the extremely unusual cases where `_STD` is needed in front of something that's the name of a class template.